### ISSUE TITLE
Invalid Host Name

### ISSUE THREAD

**Issue by Khwab-kalra** [facebookresearch/llama#387]
![Screenshot 2023-07-19 092920](https://github.com/facebookresearch/llama/assets/58007063/88908a6e-ed52-40e5-a2e7-7eeb2dfc1db5)
I have winget and md5sums installed, and even have the unique URL but still am facing this issue, while running _"bash download.sh"_.

Any help/guidance would be appreciative.

**Thanks**

**Comment by Khwab-kalra** [comment]
Yes, i've copied the link by selecting it and not using "copy Link", but it still shows this error

**Comment by fadnavismehul** [comment]
Same issue for me on windows, I fixed it by editing the download.sh script

Put this before the `read` command
`PRESIGNED_URL="https://download.llamameta..."`
Comment the line
`read -p "Enter the URL from email: " PRESIGNED_URL`

**Comment by Khwab-kalra** [comment]
> `PRESIGNED_URL="https://download.llamameta..."`

thanks a lot!! It worked

**Comment by RodLozano** [comment]
I had the same problem and it has solved it, Thanks Sir!!

**Comment by jcolano** [comment]
Worked, thanks!
